diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter6/PropertyAccessorFieldsWarnTest.kt

```diff
@@ -117,4 +117,21 @@ class PropertyAccessorFieldsWarnTest : LintTestBase(::PropertyAccessorFields) {
             LintError(24, 4, ruleId, "${WRONG_NAME_OF_VARIABLE_INSIDE_ACCESSOR.warnText()} set(value) {...")
         )
     }
+
+    @Test
+    @Tag(WarningNames.WRONG_NAME_OF_VARIABLE_INSIDE_ACCESSOR)
+    fun `shouldn't be triggered when there's a method with the same name as the property`() {
```

Comment:
Please extend tests, such cases could exist as for function extension
and for class methods
```kotlin
val String.blaBla: String
    get() = "bla".blaBla("bla")


fun String.blaBla(string: String): String = this + string

class A {
    val foo: Int
        get() = foo()

    fun foo(): Int = 15
}

val A.foo: Int
    get() = 1
```

Reply:
Also as you check for now all functions from file, make sure, that such case is valid 
```kotlin
class A {

   val String.blaBla: String
  get() = "bla".boo("bla")
 
   fun boo(string: String): String = string + string
}
```

Reply:
fixed